docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -508,43 +477,48 @@ The default is 30303.
 !!!note
     This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#exposing-ports). 
 
-### rinkeby
+### genesis-file
+
+Genesis file is used to create a custom network.
+
+!!!tip
+    If you need to use one of the well-known networks like Ethereum MainNet, you should use
+    the [`--network`](#network) option instead as all nodes of a same network have to define the 
+    exact same genesis parameters.
```

Comment:
Suggested reword: 
To use a public Ethereum network such as Rinkeby, use the [`--network`](#network) option. The network option defines the genesis file for public networks.

Reply:
Will reorder also the option as it has to be in the alphabetical order after data-path and before host-whitelist